platform-tests/go.mod

```diff
@@ -10,7 +10,9 @@ require (
 	github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac // indirect
 	github.com/cloudfoundry-community/go-cfclient v0.0.0-20180906143114-149e140bbeec
 	github.com/cloudfoundry-incubator/cf-test-helpers v1.0.1-0.20190515214722-ff434af77d37
```

Comment:
A second version of `github.com/cloudfoundry-incubator/cf-test-helpers` is added in this commit. No idea if that matters but thought it's worth highlighting.

Reply:
This might have been during a rebase